docs/user_guide/labels_and_annotations_in_yunikorn.md

```diff
@@ -22,33 +22,37 @@ under the License.
-| [DEPRECATED] `namespace.max.cpu`    | Replaced with ``namespace.quota`` since version 1.2.0                                                                                                                                                                                                                                                                    |
-| [DEPRECATED] `namespace.max.memory` | Replaced with `namespace.quota` since version 1.2.0                                                                                                                                                                                                                                                                      |
-| `namespace.enableYuniKorn`          | Controls which namespaces will have pods forwarded to Yunikorn for scheduling. Further details can be found in the [Service Configuration #admission-controller-filtering-settings](service_config#admission-controller-filtering-settings)documentation.                                                                |
-| `namespace.generateAppId`           | Controls which namespaces will have pods labeled with an `applicationId`. Further details can be found in the [Service Configuration #admission-controller-filtering-settings](service_config#admission-controller-filtering-settings) documentation.                                                                    |
+
+| Name                                                        | Description                                                                                                                                                                                                                                                                                            |
+|-------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [Classic]<br/>`yunikorn.apache.org/app-id`                  | It is equivalent to the label `yunikorn.apache.org/app-id`, and we recommend using it.                                                                                                                                                                                                                 |
+| [Classic]<br/>`yunikorn.apache.org/queue`                   | It is equivalent to the label `yunikorn.apache.org/queue`, and we recommend using it.                                                                                                                                                                                                                  |
```

Comment:
Let's switch from "[Classic] to [LEGACY], and reword descriptions as follows:

Equivalent to the preferred label `yunikorn.apache.org/queue`.